src/aws.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable camelcase */
+import AWS from 'aws-sdk'
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION || 'us-west-2',
+  apiVersions: {
+    lambda: '2015-03-31',
+    s3: '2006-03-01'
+  }
+})
+
+const bucketName = process.env.AWS_BUCKET || 'sinopia-marc-development'
+const lambdaName = process.env.AWS_RDF2MARC_LAMBDA || 'sinopia-rdf2marc-development'
+
+export const requestMarc = (resourceUri, resourceId, username, timestamp) => {
+  // These are tied to the user so that can list all of the MARC records requested by a user.
+  const marcKey = `${username}/${resourceId}/${timestamp}`
```

Comment:
Instead of using S3 as a way to track the MARC records requested per user, could we track this info in MongoDB? (I realize this out of scope for this PR but since we are planning to add personalization support I wonder if storing username in the path is the best mechanism for this functionality)

Reply:
So I haven't fully thought this out, but the S3 files are set to expire after 30 days. This gets us out of the file cleanup business, but wouldn't play well with Mongo recordkeeping.

That being said, if we decided to go the Mongo route, this could easily be removed. And yes, this is a premature optimization so if you'd like it removed now, I'm OK with it.

Reply:
Hmm, on second thought let's keep this in because like you mentioned, the 30-day S3 clean-up would mess-up MongoDB record keeping. The less we have coordinate the better at this point.